src/cloud-api-adaptor/test/provisioner/trustee_kbs.go

```diff
@@ -393,7 +393,14 @@ func getNodeIPForSvc(deploymentName string, service corev1.Service, cfg *envconf
 	return "", fmt.Errorf("Node IP not found for Service %s", service.Name)
 }
 
-func (p *KeyBrokerService) GetKbsEndpoint(ctx context.Context, cfg *envconf.Config) (string, error) {
+func (p *KeyBrokerService) GetCachedKbsEndpoint() (string, error) {
```

Comment:
This function relies on `GetKbsEndpoint` having been called at provisioning time to set the endpoint field?

Reply:
Yes, GetCachedKbsEndpoint will use the value set in GetKbsEndpoint, where GetKbsEndpoint will be initialized in main both in e2e test and provisioner.